config/application.yml.example

```diff
@@ -67,6 +67,8 @@ use_dashboard_service_providers: 'false'
 dashboard_url: 'https://dashboard.demo.login.gov'
 valid_authn_contexts: '["http://idmanagement.gov/ns/assurance/loa/1", "http://idmanagement.gov/ns/assurance/loa/3"]'
 
+usps_upload_sftp_timeout: '15'
```

Comment:
Thanks for adding the timeout. May I suggest making it lower? Maybe 5 seconds? It should at least be lower than the default Rack::Timeout setting, which is 15 seconds.